semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SemanticdbPipeline.scala

```diff
@@ -28,7 +29,7 @@ trait SemanticdbPipeline extends SemanticdbOps { self: SemanticdbPlugin =>
   }
 
   def handleCrash(unit: Option[g.CompilationUnit]): PartialFunction[Throwable, Unit] = {
-    case ex: Throwable =>
+    case NonFatal(ex) =>
```

Comment:
Your raised a good question about why metac defaults to warnings instead of errors. I think it would be good to change the default to error with v4. You in favor of that change?

Reply:
Yes. I won't have time to get this through myself this week, but if you do, that'd be great.

Reply:
I may take a stab at it, opened https://github.com/scalameta/scalameta/issues/1742